build_projects/dotnet-cli-build/DebTargets.cs

```diff
@@ -7,15 +7,25 @@
 using Microsoft.DotNet.InternalAbstractions;
 
 using static Microsoft.DotNet.Cli.Build.Framework.BuildHelpers;
+using Microsoft.Build.Utilities;
 
 namespace Microsoft.DotNet.Cli.Build
 {
-    public class DebTargets
+    public class DebTargets : Task
     {
+        public override bool Execute()
+        {
+            BuildContext context = new BuildSetup("MSBuild").UseAllTargetsFromAssembly<DebTargets>().CreateBuildContext();
+            BuildTargetContext c = new BuildTargetContext(context, null, null);
+
+            return GenerateDebs(c).Success;
+        }
+
         public static BuildTargetResult GenerateDebs(BuildTargetContext c)
         {
             if (CurrentPlatform.IsPlatform(BuildPlatform.Ubuntu))
             {
+                PrepareTargets.Init(c);
```

Comment:
The point of leaving this mostly as is was to prevent the need to do that in this turn of the crank. In the next turn of the crank these won't be tasks at all. 

If you're saying that should happen before this goes in that is fine with me, just will take a little bit of time.